### DISCUSSION TITLE
Is there a date for version 0.12.0 release?

### DISCUSSION THREAD

**Discussion by aryeko** [hbldh/bleak#569] [Resolved discussions] [answered]
Hi,

I saw a few recent interesting commits I would like to benefit from. For now I'm getting the develop branch but it's a bit complicated to deal with, mainly due to package managers.

I just wanted to know if you have a date for the next release @hbldh ?

Thank you

**Comment by hbldh** [comment]
I have been completely occupied with a lot of other things for quite some time, so I am not very knowledgable about the current state of the `develop` branch. @dlech might be able to give a better estimate on this. There are several PRs open than I think might be worth to wait for...

**Reply by dlech** [reply]
Yes, I've been thinking it is time for a release. @hbldh, do you have time/want to do any testing in the next week or two before a release? I will probably be able to get the non-blocked pull requests merged in the next day or two.

**Comment by dlech** [comment]
FYI, https://github.com/hbldh/bleak/pull/575

**Comment by egnor** [comment, marked as answer]
At your convenience, I'd love a 0.12.2 (?) to pick up https://github.com/hbldh/bleak/pull/621 which was affecting me. I'll source from github for now... but... thanks! :pray:

**Reply by hbldh** [reply]
I have been extremely absent from all things Bleak for a while. @dlech where do we stand right now? I have a PR I would like to merge before release, but it might be better to push that to 0.13?

**Reply by dlech** [reply]
If we wait a few weeks, we could get some of the PRs merged and add Python 3.10 support (official release date 2021-10-04) and do a 0.13 release.
